sycl/test-e2e/ESIMD/assert.cpp

```diff
@@ -2,11 +2,6 @@
 
 // RUN: %{build} -DSYCL_FALLBACK_ASSERT=1 -o %t.out
 // RUN: %{run} %t.out &> %t.txt ; FileCheck %s --input-file %t.txt
-//
-// The test still fails after GPU driver update on Linux. Temporary marking it
-// as expected to fail, whilst it is being investigated, see intel/llvm#11359
```

Comment:
I did not understand what happened. Did you?
It seems the GPU driver was updated and it caused regression. Then this test was disabled. But not it reports pass with same updated GPU driver.
Is the issue really fixed, or the test is flaky?

If the issue mentioned in this comment is really fixed, please add a corresponding comment in the description of the patch. Something like 'closes #11359'

Reply:
It is passing on Gen12, and is flaky on PVC. I updated REQUIRES line.